website/docs/d/scc_posture_scans_summary.html.markdown

```diff
@@ -0,0 +1,93 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_scc_posture_scan_summary"
+description: |-
+  Get information about scans_summary
+subcategory: "Security and Compliance Center"
+---
+
+# ibm_scc_posture_scan_summary
+
+Provides a read-only data source for scans_summary. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax.
```

Comment:
1. Review information of Security and Compliance Center posture scan summary. 
2. Add reference link for posture scan summary